src/components/attendee/ProfileOverlay/ProfileOverlay.tsx

```diff
@@ -162,6 +169,16 @@ export const ProfileOverlay: React.FC<ProfileOverlayProps> = ({ profile }) => {
     ]
   );
 
+  const history = useHistory();
+  const { worldSlug, spaceSlug } = useWorldAndSpaceByParams();
+
+  const [{ loading: isLoggingOut }, logout] = useAsyncFn(async () => {
+    await firebase.auth().signOut();
+    onOverlayClose();
+
+    history.push(generateAttendeeSpaceLandingUrl(worldSlug, spaceSlug));
```

Comment:
I think this `generateAttendeeSpaceLandingUrl` is (or should be) marked as `@deprecated` in favor of `generateUrl()`.
Can you please do  the replacement?

The idea is that once we remove all these and only `generateUrl()` remains, we can improve that one for nicer DX and more robust handling.

Reply:
done: https://github.com/sparkletown/sparkle/pull/3003/commits/07ec924f308486ffff85bb746fdba69810c19e56